terraform/modules/quorum-cluster-region/quorum-cluster-region-backup-event.tf

```diff
@@ -0,0 +1,399 @@
+
+# This is to be placed in quorum-cluster-region
+
+provider "archive" {}
+provider "random" {}
+provider "null" {}
```

Comment:
nit: Double check that you need the null provider here and can't just inherit it. I know I've used it but it may or may not be in this module.